src/puppetlabs/puppetdb/http/resources.clj

```diff
@@ -16,7 +16,8 @@
                   version
                   (params "query")
                   paging-options
-                  (:scf-read-db globals)))
+                  (:scf-read-db globals)
+                  (:url-prefix globals)))
                http-q/restrict-query-to-active-nodes)}))
```

Comment:
also here -- I think this will cause people querying catalogs to get invalid hrefs if their nodes are deactivated.